src/compas/datastructures/mesh/planarisation_numpy.py

```diff
@@ -1,128 +0,0 @@
-from __future__ import print_function
-from __future__ import absolute_import
-from __future__ import division
-
-from compas.geometry import project_points_plane
-from compas.geometry import centroid_points
-from compas.geometry import distance_point_point
-from compas.geometry import distance_line_line
-from compas.geometry import bestfit_plane
-
-from compas.utilities import window
-
-
-__all__ = [
-    'mesh_flatness',
-    'mesh_planarize_faces',
-]
-
-
-def mesh_flatness(mesh, maxdev=1.0):
```

Comment:
I'm not sure here... these two functions in this file are deleted, but still in the imports, so, I guess they are somewhere else and this was duplicated?

Reply:
no they are in `planarisation.py` the numpy version was just a duplicate that was never updated to be based on numpy... so this doesn't break anything afaik...